web/models.py

```
@@ -3046,3 +3046,85 @@
+        }
+
+class VirtualClassroomWhiteboard(models.Model):
+    """Model to store whiteboard data for each virtual classroom"""
+    classroom = models.OneToOneField(VirtualClassroom, on_delete=models.CASCADE, related_name='whiteboard')
+    canvas_data = models.JSONField(default=dict, blank=True)
+    background_image = models.TextField(blank=True, null=True)
+    last_updated = models.DateTimeField(auto_now=True)
+    created_at = models.DateTimeField(default=timezone.now)
+    last_updated_by = models.ForeignKey(User, on_delete=models.SET_NULL, null=True, blank=True)
+
+    def __str__(self):
```

Ruff (0.11.9)

Missing return type annotation for special method `__str__`

Add return type annotation: `str`

(ANN204)